tutorial/README.md

```diff
@@ -0,0 +1,413 @@
+val salesDF = sqlContext.read
+             .format("com.databricks.spark.redshift")
+             .option("url",jdbcURL ) 
+             .option("tempdir", tempS3Dir) 
+             .option("query", salesQuery)
+             .load()
+salesDF.show()
+```
+Listing 3: Reading from a Redshift query
+
+The lines in Listing 3 will execute to produce the following output
+
+```{r, engine='bash'}
++-------+------+--------+-------+-------+------+-------+---------+----------+
+|salesid|listid|sellerid|buyerid|eventid|dateid|qtysold|pricepaid|commission|
++-------+------+--------+-------+-------+------+-------+---------+----------+
+| 102373|117133|   26190|  35491|   2141|  2191|      4|  1008.00|    151.20|
+|  75861| 86640|   43402|  39545|   8372|  2191|      2|   372.00|     55.80|
+|  56302| 63546|    5760|   5797|   1489|  2191|      2|   372.00|     55.80|
+|  83603| 95341|   27027|  27881|   6034|  2191|      1|   288.00|     43.20|
+|  40652| 45468|   27557|  28366|   5099|  2191|      2|   510.00|     76.50|
+| 157586|206655|   13213|  47394|   3665|  2191|      2|  4018.00|    602.70|
+|  99761|114026|    7466|  21189|   5621|  2191|      2|   102.00|     15.30|
+|  68467| 77797|   49538|  27029|   1489|  2191|      2|   188.00|     28.20|
+|  66470| 75420|   22851|  37849|   7604|  2191|      2|   420.00|     63.00|
+|  36700| 40838|   15099|  45514|   8027|  2191|      4|  1400.00|    210.00|
+| 126896|145346|   18243|  10273|   6783|  2191|      2|   866.00|    129.90|
+|  55416| 62529|   42223|  26273|   1489|  2191|      4|   532.00|     79.80|
+|  48503| 54812|   42828|  12282|   8372|  2191|      2|    80.00|     12.00|
+|   3737|  4035|   11666|  23692|   6034|  2191|      2|   302.00|     45.30|
+| 154069|178359|    3897|   5815|   8168|  2191|      1|   290.00|     43.50|
+|  37817| 42124|   24592|  26543|   6783|  2191|      4|  1300.00|    195.00|
+|    924|   933|   48898|  25710|   1594|  2191|      3|   504.00|     75.60|
+| 112232|128585|   30390|  44375|    914|  2191|      1|   126.00|     18.90|
+|  97099|110826|   24730|  32758|   3331|  2190|      1|   319.00|     47.85|
+|  74902| 85497|   45049|   2318|   7547|  2190|      2|   554.00|     83.10|
++-------+------+--------+-------+-------+------+-------+---------+----------+
+only showing top 20 rows
+```
+
+### Under the hood - Load Function ###
+
+In this section we will take a peek inside spark-redshift to understand how exactly the LOAD function works. Specifically we will look at how `event` table was read into a DataFrame. The LOAD is a two step process
+
+1. [UNLOAD](http://docs.aws.amazon.com/redshift/latest/dg/r_UNLOAD.html) data from the Redshift table to S3
+2. Consume the unloaded files in S3 via a custom InputFormat into an RDD which is then wrapped as a DataFrame instance using the schema obtained from Redshift.
+
+#### UNLOAD Redshift to S3 ####
+
+Figure 1 shows the the actions required to perform this step.
```

Comment:
Read out of context, this sentence doesn't make sense, since "this step" is ambiguous.

I'd prefer to be more explicit here and say something like "Figure 1" or "The following diagram shows the steps that are performed when Spark Redshift UNLOADs data from Redshift to S3." That sentence makes sense even if the reader didn't look at the section header.